src/commands/bash_hook.rs

```diff
@@ -186,11 +192,17 @@ pub fn execute<'a>(matches: &ArgMatches<'a>) -> Result<(), Error> {
             "___SENTRY_TRACEBACK_FILE___",
             &traceback.display().to_string(),
         )
-        .replace("___SENTRY_LOG_FILE___", &log.display().to_string())
-        .replace(
+        .replace("___SENTRY_LOG_FILE___", &log.display().to_string());
+
+        if matches.is_present("cli") {
+            script = script.replace(
             "___SENTRY_CLI___",
-            &env::current_exe().unwrap().display().to_string(),
-        );
+            matches.value_of("cli").unwrap()
+            );
+        } else {
+            script = script.replace("__SENTRY_CLI__", &env::current_exe().unwrap().display().to_string());
+        }
```

Comment:
Instead of this, I tried every possible variation of `matcher` and `unwrap_or_else` etc. but they gave me hell with the temporary return value of the `env::current_exe().unwrap().display().to_string()` call or some weird mismatch between `String` and `str` types.

Open to suggestions to make this more "rustic".

Reply:
```rust
.replace(
  "__SENTRY_CLI__",
  matches
    .value_of("cli")
    .unwrap_or(&env::current_exe().unwrap().display().to_string()),
);
```

Reply:
@kamilogorek that runs the expression `&env::current_exe().unwrap().display().to_string()` for the fallback unconditionally, which is why I went with this path (after exhausting the path of closures)

Reply:
True, that'll be eagerly evaluated, and returning a ref from `unwrap_or_else` is tricky.